src/components/RecipeCard/RecipeCard.styled.tsx

```diff
@@ -0,0 +1,229 @@
+  padding: 30px;
+  background: radial-gradient(
+    circle,
+    ${({ theme }) => theme.palette.secondary.main},
+    ${({ theme }) => theme.palette.primary.light}
+  );
+  border-radius: 100%;
+  width: 100px;
+  height: 100px;
+  box-shadow: 1 -1px -1px 5px 3px rgba(0, 0, 0);
+  font-size: 30px;
+  font-family: "Arial", sans-serif;
+  color: ${({ theme }) => theme.palette.primary.dark};
+
+  ${({ theme }) => theme.breakpoints.down("lg")} {
+    width: 80px;
+    height: 80px;
+  }
+  ${({ theme }) => theme.breakpoints.down("md")} {
+    width: 100px;
+    height: 100px;
+  }
+  ${({ theme }) => theme.breakpoints.down("sm")} {
+    width: 80px;
+    height: 80px;
+  }
+`;
+
+export const StyledBodyContainer = styled("div")`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+
+  ${({ theme }) => theme.breakpoints.down("md")} {
+    flex-direction: column;
+    align-items: center;
+  }
+`;
+
+export const StyledGalleryContainer = styled("div")`
+  width: 100%;
+  height: inherit;
+  padding: 0;
+  box-sizing: border-box;
+  border: 6px solid;
+  border-color: ${({ theme }) => theme.palette.primary.main};
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.1);
+  border-radius: 8px;
+
+  margin: ${({ theme }) => theme.spacing(2)};
+  ${({ theme }) => theme.breakpoints.down("md")} {
+    margin: ${({ theme }) => theme.spacing(2)} 0;
+  }
+`;
+
+export const StyledGalleryImag = styled("img")`
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  object-position: center;
+  display: block;
+`;
+
+export const StyledEntryContainer = styled("div")`
+  padding: ${({ theme }) => theme.spacing(2)};
+  font-family: "Playfair Display", serif;
+  font-size: 22px;
+  ${({ theme }) => theme.breakpoints.down("md")} {
+    font-size: 20px;
+  }
+  ${({ theme }) => theme.breakpoints.down("sm")} {
+    font-size: 16px;
+  }
+`;
+
+export const NarrowViewContainer = styled("div")`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  box-sizing: border-box;
+  padding: 0 ${({ theme }) => theme.spacing(2)};
+`;
+
+export const RecipeCardContainer = styled("div")`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+`;
+
+export const RowOfWideViewContainer = styled("div")`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+`;
+
+export const StyledDivider = styled(Divider)`
+  height: 2px;
+  background-color: ${({ theme }) => theme.palette.primary.dark};
+  width: 100%;
+  margin-bottom: ${({ theme }) => theme.spacing(4)};
+  ${({ theme }) => theme.breakpoints.down("lg")} {
+    margin-bottom: ${({ theme }) => theme.spacing(3)};
+  }
+  ${({ theme }) => theme.breakpoints.down("md")} {
+    margin-bottom: ${({ theme }) => theme.spacing(2)};
+  }
+  ${({ theme }) => theme.breakpoints.down("sm")} {
+    height: 1px;
+    margin-bottom: ${({ theme }) => theme.spacing(1)};
+  }
+`;
+
+export const StyledSmallDivider = styled(Divider)`
+  height: 1px;
+  background-color: ${({ theme }) => theme.palette.primary.dark};
```

Comment:
```suggestion
export const StyledSmallDivider = styled(Divider)(({ theme }) =>`
  height: 1px;
  background-color: ${theme.palette.primary.dark};
```
It would be more elegant to use single function instead of multiple with theme injection. Please change it in all styled files